andela/activo-api
# #158286547 Add tests for dynamic filter

#### What does this PR do?
This PR adds in tests for the the dynamic filter functionality

#### Description of Task to be completed?
* Add tests for the checks the functionalities of dynamic filter.

#### How should this be manually tested?
* Run `git fetch origin ch-dynamic-filter-tests-158286547:ch-dynamic-filter-tests-158286547`, then `git checkout ch-dynamic-filter-tests-158286547`
* Run the command `pytest tests/test_query_filter.py`, to see the test results.


#### Any background context you want to provide?
None

#### What are the relevant pivotal tracker stories?
[#158286547](https://www.pivotaltracker.com/story/show/158286547)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
